install/0000_30_machine-api-operator_09_rbac.yaml

```
@@ -332,6 +332,21 @@ subjects:
     name: prometheus-k8s
     namespace: openshift-monitoring
 
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: cloud-provider-configmap-reader
+  namespace: openshift-machine-api
```

The rolebinding needs to be in the namespace that the role is in, so this needs to be `openshift-config`
```suggestion
  namespace: openshift-config
```